src/jsxc.lib.gui.js

```diff
@@ -2050,7 +2050,12 @@ jsxc.gui.window = {
       win.find('.jsxc_sendFile').click(function() {
          $('body').click();
 
-         jsxc.gui.window.sendFile(bid);
+         if(typeof jsxc.options.get('httpupload') !== "undefined"){
```

Comment:
@sualko agreed, at least as long as the files and their filenames aren't stored encrypted on servers with http_upload support